### ISSUE TITLE
is any plan to integrate typedunits into labrad?

### ISSUE THREAD

**Issue by kylinzzx** [quantumlib/TypedUnits#25]
typedunits is much faster than labrad.units. is any plan to replace labrad.units with typedunits?
thank you.

**Comment by NoureldinYosri** [comment]
no, these are two different libraries doing similar things ... you can make your choice for which one better suits your use
